crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -438,6 +468,10 @@ pub struct PickledInboundGroupSession {
     /// The algorithm of this inbound group session.
     #[serde(default = "default_algorithm")]
     pub algorithm: EventEncryptionAlgorithm,
+    /// Flag remembering if the session is safe. Reflects the fact that we are
```

Comment:
I haven't reviewed this yet in its entirety, but I really think we want to avoid introducing more term overloads. So we should choose either the "safe/unsafe" or "trusted/untrusted" terminology and stick with it everywhere.

And given that `trusted` is already a thing that exists in the spec and is what is suggested in MSC3879, we'll probably need to side with that.

Reply:
Updated it as per https://github.com/matrix-org/matrix-rust-sdk/issues/1366